src/add_lable.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+from dotmap import DotMap
+from github import Github
+from github import Auth
+from github.Repository import Repository
+
+ORIGINAL_ISSUES_FILE = "../res/issues.json.zip"  # ../res/ o res/
+LABELED_ISSUES_FILE = "../res/labeled_issues.json"
+SECONDS_TO_LABEL = 60  # Secondi entro i quali l'etichetta deve essere stata aggiunta
+REPOSITORY = "microsoft/vscode"
+AUTH_TOKEN = "INSERT_YOUR_TOKEN"
+
+repo: Repository
+testing = False  # False to run all the issues, otherwise the number of firsts issues to run
+
+
+def init_github():
+    auth = Auth.Token(AUTH_TOKEN)
+    g = Github(auth=auth)
+    global repo
+    repo = g.get_repo(REPOSITORY)
+
+
+def get_creation_labels(issue_id):
```

Comment:
Add type annotation and the pydoc.
Here a generated example for reference:
```python
def get_creation_labels(issue_id: int) -> str:
    """
    Retrieve the labels that were added to a GitHub issue within 60 seconds of its creation.

    Args:
        issue_id (int): The unique identifier of the issue in the repository.

    Returns:
        str: A string containing the names of the labels that were added to the issue shortly after its creation, 
             separated by spaces.
    """
    issue: Issue = repo.get_issue(number=issue_id)
    events = issue.get_timeline()
    created_labels: List[str] = []
    for event in events:
        if event.event == "labeled":
            if ((event.created_at - issue.created_at).total_seconds()) < 60:
                label_name = event.raw_data.get("label", {}).get("name")
                created_labels.append(label_name)

    return " ".join(created_labels)
```

It is important to understand:
1. the usage of the method
2. the arguments and their type
3. the expected return and its type